clang/include/clang/Tooling/DependencyScanning/DependencyScannerImpl.h

```diff
@@ -1,4 +1,4 @@
-//===- DependencyScanner.h - Performs module dependency scanning *- C++ -*-===//
+//===- DependencyScannerImpl.h - Implements dependency scanning *- C++ -*--===//
```

Comment:
`Impl.h` is usually not a public header name. It is a header that we put inside `lib/...` for implementation details.

I actually don't think we expose this class in dependency scanning APIs (other than a pointer?) so maybe it is better to move it into `lib/Tooling/DependencyScanning`